django/events/templates/events/events_list.html

```diff
@@ -58,6 +58,31 @@ <h2>{{event.title}}</h2>
       {% endfor %}
     </ul>
   </div>
+
+<script>
+function month_int_to_litteral(month_number){
+    const month = ['Janvier', 'Février', 'Mars', 'Avril', 'Mai', 'Juin', 'Juillet', 'Août', 'Septembre', 'Octobre', 'Novembre', 'Décembr'];
+    return month[month_number]
+}
+
+function formatDate(){
+    var today = new Date();
+    day = today.getDate();
+    console.log(today);
+    month = today.getMonth();
+    year = today.getFullYear();
+ 
+    today = day + " " + month_int_to_litteral(month) + " " + year;
+    return today;
+}
+
+
+var date_p = document.getElementById("date");
+date_p.innerHTML = formatDate();
```

Comment:
Pourquoi ne pas utiliser Vue à la place pour interagir avec le DOM ? Si vous avez déjà VueJS dans l'appli, autant en profiter :slightly_smiling_face: 

 ```suggestion
var date_p = new Vue({ el: '#date', data: { today: formatDate() }  });
```

Et remplacer l'élément du DOM par:

```diff
- <p id="date"></p>
+ <p id="date" v-text="today"></p>
```

Cf [exemple ici](http://jsfiddle.net/z13kaodw/)